plugins/scaffolder-react/src/next/components/Stepper/Stepper.tsx

```diff
@@ -128,6 +137,13 @@ export const Stepper = (stepperProps: StepperProps) => {
     [setFormState],
   );
 
+  const handleCreate = useCallback(() => {
+    props.onCreate(formState);
+    const name =
+      typeof formState.name === 'string' ? formState.name : undefined;
+    analytics.captureEvent('create', name ?? props.templateName ?? 'unknown');
+  }, [props, formState, analytics]);
```

Comment:
`props` as a dependency looks problematic to me. doesn't this mean `useCallback` is basically triggered with any prop (except layouts & compoents) changing? Alternatives I could see is either use `props.onCreate` as a dependency or pass `onCreate` as a parameter (not sure the 2nd option is great either, but would expect `onCreate` to not change)

Reply:
It would mean that the function gets re-defined when those props change yes, so it's a little inefficient but it's not calling it when the props change.